crates/nargo_cli/src/cli/codegen_verifier_cmd.rs

```diff
@@ -19,8 +21,22 @@ pub(crate) fn run(args: CodegenVerifierCommand, config: NargoConfig) -> Result<(
     #[allow(deprecated)]
     let smart_contract_string = backend.eth_contract_from_cs(compiled_program.circuit);
 
-    let contract_dir = config.program_dir.join(CONTRACT_DIR);
-    create_named_dir(&contract_dir, "contract");
+    let manifest_path = config.program_dir.join(PKG_FILE);
+    if !manifest_path.exists() {
+        return Err(CliError::MissingTomlFile("Nargo".to_owned(), manifest_path));
+    }
```

Comment:
```suggestion
    let manifest_path = config.program_dir.join(PKG_FILE);
```

We don't need to check for existence of the manifest here as that's done at CLI startup.